end_to_end/test_llama2_13b.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
```

Comment:
Why two scripts? They seem duplicative?

Reply:
the *_chkpt.sh does checkpoint conversions only, one from meta to maxtext, and one from scanned to unscanned. 

The generate_param_only_checkpoint takes a diff arg when the checkpoint is from pretrained/finetuned than its converted directly from meta. This was the error both 13b and 70b encountered during unroll conversion. Also, after the unroll conversion, added another check for golden output.

Happy to remove the *_chkpt.sh altogether, or add the unroll conversion for meta converted chkpt and decode output validation in the main script.

Reply:
I think this flow would be better if we documented the full user journey for llama-13b in one place. Happy to troubleshoot the bug.

(Also, can you think about what user facing changes we'd want to make this usable? For example, I think you're targeting a v4-16. Can you document that?)

Reply:
Updated the testing script with more comment, can also provide a readme for the 13b

Reply:
Updated the testing script with more comment, I can also write a 13b readme, once confirm the issue when using v4-16 vs v4-128